.github/workflows/ci.yaml

```diff
@@ -69,21 +69,21 @@ jobs:
         bzlmodEnabled: [true, false]
         exclude:
           # macos is expensive (billed at 10X) so don't test these
-          - os: macos-latest 
+          - os: macos-latest
             folder: e2e/custom_registry
-          - os: macos-latest 
+          - os: macos-latest
             folder: e2e/wasm
-          - os: macos-latest 
+          - os: macos-latest
             folder: e2e/crane_as_registry
-          - os: macos-latest 
+          - os: macos-latest
             folder: e2e/pull
           - os: macos-latest
-            bazelversion: 5.3.2
+            bazelversion: 5.4.0
           # e2e/pull is bzlmod only but it has test for both cases.
           - folder: e2e/pull
             bzlmodEnabled: false
           # Don't test bzlmod with Bazel 5 (not supported)
-          - bazelversion: 5.3.2
+          - bazelversion: 5.4.0
```

Comment:
I have no idea why this is happening but here's the error https://github.com/bazel-contrib/rules_oci/actions/runs/8638197772/job/23682118018?pr=541#step:8:37

Reply:
that error looks clear that rules_go is enforcing some policy. The bump is fine